File: add_element.py
arr=[]
arrLen=int(input("Enter array size: "))
print(f"Enter {arrLen} elements:")
for _ in range(arrLen):
  ele=int(input())
  arr.append(ele)
newEle=int(input("Enter new element: "))
arr.append(newEle)
print("New array: ")
print(arr)

File: add_element.cpp
#include<stdio.h>
int main()
{
    int arr[200], arrLen, i, j;
    printf("Enter number of elements: ");
    scanf("%d",&arrLen);
    printf("Enter %d elements\n",arrLen);
    for(i=0; i<arrLen; i++){
        scanf("%d",&arr[i]);
    }
    printf("Enter the new element: ");
    scanf("%d",&arr[arrLen]);
    arrLen++;
    printf("New Array:\n");
    for(j=0; j<arrLen; j++){
        printf(" %d ",arr[j]);
    }
    return 0;
}
